railties/lib/rails/commands/server/server_command.rb

```diff
@@ -112,12 +112,18 @@ class ServerCommand < Base # :nodoc:
         desc: "Specifies the PID file."
       class_option "dev-caching", aliases: "-C", type: :boolean, default: nil,
         desc: "Specifies whether to perform caching in development."
+      class_option "log-stdout", aliases: "-l", type: :boolean, default: nil,
+        desc: "Specifies whether to output Rails' log to std out (true = always, false = never, default = in development). Running as a Daemon always disables output."
 
       def initialize(args = [], local_options = {}, config = {})
         @original_options = local_options
         super
         @server = self.args.shift
-        @log_stdout = options[:daemon].blank? && (options[:environment] || Rails.env) == "development"
+        @log_stdout =
```

Comment:
Maybe this should be moved to a memoized private function named something like `log_to_stdout?`? I think with the logic change this makes sense.

Reply:
Seems good. I wouldn't even memoize it. It's not a candidate for being called thousands of times, so the 2-3 calls in a server boot are fine to redo the work.

Reply:
Alright, removed the memoizing.